orc8r/gateway/go/services/bootstrapper/service/rpc.go

```diff
@@ -101,7 +102,7 @@ func (b *Bootstrapper) getGrpcOpts(useProxy bool, cfg *config.ControlProxyCfg) [
 			}
 		} else {
 			tlsCfg = &tls.Config{
-				InsecureSkipVerify: true,
+				InsecureSkipVerify: cfg.CloudConnectionSkipVerify,
```

Comment:
func (b *Bootstrapper) getGrpcOpts(useProxy bool, cfg *config.ControlProxyCfg) ([]grpc.DialOption, error) {
...
...
...
} else if cfg.CloudBootstrapperSkipVerify {
    tlsCfg = &tls.Config{
				InsecureSkipVerify: true ,
    }
} else {
    return opts, fmt.Errorf("no root CA to verify server authenticity")
}
...